src/components.js

```diff
@@ -125,4 +126,14 @@ export default {
             name: 'caret',
         },
     },
+    Slider: {
```

Comment:
Also make sure you add new components to this file and properly set `defaultProps`. You can then test this component by running `npm run dev`. Right now this doesn't compile because `@extremely-dark-gray` is missing from `variables.less` file.